### PR TITLE
Fixed bible names and added synonyms for 🇧🇷

### PR DESCRIPTION
**Pull request by hebertialmeida**

### CODE REVIEW

File: bibles/pt/arc/info.js [hunk 1]
```diff
@@ -6,119 +6,142 @@ var info = {
...
+				"Rt"
 			]
 		},
 		{
 			"name": "1 Samuel",
 			"numChapters": 31,
 			"synonyms": [
-				"1 Samuel"
+				"1 Samuel",
+				"1 Sm",
```
Comment (VitalikL) [inline bibles/pt/arc/info.js]: @hebertialmeida Technically you do not need to provide synonyms without space as the helper function does it automatically

Comment (hebertialmeida) [inline bibles/pt/arc/info.js]: Oh, the standard abbreviation is `1Sm` but I added with space `1 Sm`, do you think we should remove that?

**Comment by VitalikL** [comment]
@hebertialmeida Actually other way around. So "1 Sm" will be converted to "1 ?Sm" so space is optional see https://github.com/Adventech/bible-tools/blob/master/bible_helpers.js#L48

**Comment by hebertialmeida** [comment]
Better to clean up so.

**Comment by hebertialmeida** [comment]
Done

**Pull request merged**